src/main.cpp

```diff
@@ -316,7 +321,7 @@ int main(int argc, char *argv[]) {
  * choosing xdot to be a vector of all ones, ybar = 1.0;
  * ==================================================================================*/
  // TODO: Why is this disabled?
```

Comment:
I am pretty sure that the below tests don't work anymore. But somehow I'd want to keep them, as a template, for including them later again...

Reply:
If you agree, I would park them in another branch and get them out of master.

Reply:
Yes I do agree!